strapi/strapi
# Entity service data param for create and update

### What does it do?

Adds types for the data param for the create and update functions of the entity service.

### Why is it needed?

To improve the typings when creating and updating content types from the entity service.

### How to test it?
Generate types for a project with the `yarn strapi ts:generate` command and then go to packages/core/strapi/lib/services/entity-service/types/params/index.d.ts and write something like:
```
const h: Data.Input<"schema-uid"> = {
  // type suggestions should happen here :)
};
```
where `schema-uid` is the UID of one of your schemas generated when you ran the ts:generate command.

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request
